### PR TITLE
Add P-R lettered article stubs from glossary

### PR DESCRIPTION
**Pull request by Pennek**
Part of organization effort - #3143
Probably the heaviest commit I've made to date. I split up the old performance points article into multiple new ones. Some of them are stubs and some of them are fully fleshed out articles. Gosh I want those hours spent back...

Adds:

- Rank
- Pass
- peppy
- Performance Points
- Performance Point System
- ppv2
- Total Performance Points
- Weighting System
- Recalculate Slider Lengths
- Resnap
- Ranking
- Restricted Mode

Modifies:
- Disambiguation
- ppv1 (added needs_cleanup tag)

and outdates a couple of translations

### CODE REVIEW

File: wiki/Performance_Points/Performance_Point_Systems/en.md
Comment (gagahpangeran) [inline wiki/Performance_Points/Performance_Point_Systems/en.md:45]: This forum link can't be opened (maybe it's in wasteland).

File: wiki/Disambiguation/Rank/en.md
Comment (paper-toaster) [inline wiki/Disambiguation/Rank/en.md:12]:
```suggestion
- Synonym for a letter grade.
```

File: wiki/Disambiguation/Rank/en.md
Comment (paper-toaster) [inline wiki/Disambiguation/Rank/en.md:14]:
```suggestion
- The verb "to rank". When someone goes through the [beatmap ranking procedure](/wiki/Beatmap_ranking_procedure), making scores achieved by anyone playing it count towards each player's player ranking.
```
`/en.md` will redirect to the english article specifically iirc

File: wiki/Pass/en.md
Comment (paper-toaster) [inline wiki/Pass/en.md:9]:
```suggestion
A **pass** desribes the completion of a [beatmap](/wiki/Beatmaps) with or without the use of mods, or the reaching of a break section with more than 50% [health](/wiki/Beatmapping/Health). In the case of reaching a break section, an indicator will display with sound to show that the player has passed the previous section.
```

File: wiki/People/peppy/en.md
Comment (paper-toaster) [inline wiki/People/peppy/en.md:10]:
```suggestion
The online alias of Dean Herbert (stylised with a lowercase "p") and is the creator of the osu! project and its primary [developer](/wiki/People/The_Team/Developers).
```
"he" just sounds really informal imo

File: wiki/Performance_Points/Performance_Point_Systems/en.md
Comment (paper-toaster) [inline wiki/Performance_Points/Performance_Point_Systems/en.md:1]: I don't think we need this; this is literally just a smaller-ish version of the actual peformance points article, which has all of this info and more.

File: wiki/Performance_Points/en.md
Comment (paper-toaster) [inline wiki/Performance_Points/en.md:1]: this article was requested to be cleaned-up, not be removed and split-up in separate sections. #3781 is already handling this

File: wiki/Ranking/en.md
Comment (paper-toaster) [inline wiki/Ranking/en.md:15]:
```suggestion
- Performance points ranking: based on [performance points](/wiki/Performance_Points).
```

File: wiki/Ranking/en.md
Comment (paper-toaster) [inline wiki/Ranking/en.md:22]:
```suggestion
The performance points ranking, score ranking, and Kudosu! ranking can be navigated to by using the `Rankings` dropdown panel at the top of the webpage and choosing the `Performance` option.
```

File: wiki/Ranking/en.md
Comment (paper-toaster) [inline wiki/Ranking/en.md:26]:
```suggestion
## Performance points ranking

*Full explanation: [Performance Points](/wiki/Performance_Points)*

The performance points ranking describes a player's skill-level in comparison to other players. This leaderboard is determined by the total number of performance points an individual player has, and can be seen on a player's profile and on the [performance leaderboards](https://osu.ppy.sh/rankings/osu/performance).
```

File: wiki/Ranking/en.md
Comment (paper-toaster) [inline wiki/Ranking/en.md:30]:
```suggestion
Score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance points ranking.
```

File: wiki/Ranking/en.md
Comment (paper-toaster) [inline wiki/Ranking/en.md:38]:
```suggestion
Spotlights rankings are time-limited leaderboards that compare players' skill on a specific set of beatmaps each spotlight season. Players who rank highly on spotlight rankings will receive specific awards as a result of there hard work.
```

File: wiki/Ranking/en.md
Comment (paper-toaster) [inline wiki/Ranking/en.md:42]:
```suggestion
The beatmap rankings compare players' scores against each other on individual beatmaps. They appear once a beatmap has been [Qualified](/wiki/Beatmaps#qualified), [Ranked](/wiki/Beatmaps#ranked), [Approved](/wiki/Beatmaps#approved) or Loved. First place rankings on these leaderboards can be seen on a user's profile page, and comes with a bit of prestige.
```

File: wiki/Recalculate_Slider_Lengths/en.md
Comment (paper-toaster) [inline wiki/Recalculate_Slider_Lengths/en.md:7]:
```suggestion
**Recalculate slider lengths** is a command under the `Timing` tab in the [beatmap editor](/wiki/Beatmap_Editor). Usage of this is suggested after a BPM change is made if the [beatmap](/wiki/Beatmaps) already has [sliders](/wiki/Hit_object/Slider).
```

File: wiki/Recalculate_Slider_Lengths/en.md
Comment (paper-toaster) [inline wiki/Recalculate_Slider_Lengths/en.md:9]:
```suggestion
This command will automatically adjust the length of every slider in the beatmap to the new BPM. Although most creators reccomend checking every slider and adjusting its length manually, due to this tool's inaccurate and systematic nature.
```

File: wiki/Resnap/en.md
Comment (paper-toaster) [inline wiki/Resnap/en.md:7]:
```suggestion
**Resnap** refers to a pair of commands under the `Timing` header in the [beatmap editor](/wiki/Beatmap_Editor). One of which applies to the entire [beatmap](/wiki/Beatmaps), and the other to the current timing section. If the beatmap has only one timing section, both commands are the same.
```

File: wiki/Resnap/en.md
Comment (paper-toaster) [inline wiki/Resnap/en.md:9]:
```suggestion
This command will automatically reposition all [hit objects](/wiki/Hit_object) on the timeline so that they fall on the closest beat division that the snap divisor allows. Usage is suggested when there has been a change in offset and/or BPM, after objects have been placed. Many creators reccomend checking every hit object and adjusting its position on the [timeline](/wiki/Beatmap_Editor/Timelines) manually due to this tool's inaccurate and systematic nature.
```

File: wiki/Restricted_Mode/en.md
Comment (paper-toaster) [inline wiki/Restricted_Mode/en.md:7]:
```suggestion
**Restricted mode** is a state that is automatically applied to accounts that are suspected of breaking the [Terms of Service](https://osu.ppy.sh/legal/terms). Submitted scores of gameplay will be hidden from the [leaderboards](/wiki/Ranking) as a result of this.
```

File: wiki/Ranking/en.md
Comment (paper-toaster) [inline wiki/Ranking/en.md:11]:
```suggestion
**Ranking** (alternatively called "leaderboards") is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.
```

**Review by MegaApplePi — APPROVED** [review]

File: wiki/Ranking/en.md
Comment (Joehuu) [inline wiki/Ranking/en.md:40]: Spotlights is proper.

```suggestion
The Spotlights rankings are time-limited leaderboards that compare players' skill on a specific set of beatmaps each [Spotlights](/wiki/Beatmap_Spotlights) season. Players who rank highly on Spotlights rankings will receive specific awards as a result of their hard work.
```

File: wiki/Ranking/en.md
Comment (Joehuu) [inline wiki/Ranking/en.md:34]: Hmm, I'm not even sure what is the official way to format this. See https://osu.ppy.sh/help/wiki/Modding/Kudosu and the profile for example. By the way, I'm leaning towards the `kudosu!` format.

```suggestion
## kudosu! ranking
```

File: wiki/Ranking/en.md
Comment (Joehuu) [inline wiki/Ranking/en.md:36]:
```suggestion
The kudosu! ranking compares users' accumulated [kudosu!](/wiki/Modding/Kudosu) and lists them accordingly. This ranking requires no playing skill as kudosu! is gained as part of the [modding process](/wiki/Modding).
```

File: wiki/Ranking/en.md
Comment (Joehuu) [inline wiki/Ranking/en.md:32]:
```suggestion
The score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance points ranking.
```

File: wiki/Recalculate_slider_lengths/en.md
Comment (Joehuu) [inline wiki/Recalculate_slider_lengths/en.md:7]:
```suggestion
**Recalculate slider lengths** is a command under the `Timing` tab in the [beatmap editor](/wiki/Beatmap_Editor). Usage of this is suggested after a [BPM](/wiki/Beatmapping/Beats_per_minute) change is made if the [beatmap](/wiki/Beatmaps) already has [sliders](/wiki/Hit_object/Slider).
```

File: wiki/Restricted_mode/en.md
Comment (Joehuu) [inline wiki/Restricted_mode/en.md:9]: Really weird that the help centre would give "more info" than the restricted mode article itself.

**Review by Joehuu — CHANGES_REQUESTED** [review]

File: wiki/Ranking/en.md
Comment (MegaApplePi) [reply wiki/Ranking/en.md:34]: ok, I will note this as well

File: wiki/Restricted_mode/en.md
Comment (MegaApplePi) [reply wiki/Restricted_mode/en.md:9]: This was a glossary term, but since it may cause some confusion/messy linking, I'll just remove this article instead.

Comment (Joehuu) [reply wiki/Restricted_mode/en.md]: I would've opposed removing it until I saw https://osu.ppy.sh/help/wiki/Help_Centre/Account_Restrictions. Doesn't seem needed anymore.

File: wiki/Ranking/en.md
Comment (TPGPL) [inline wiki/Ranking/en.md:13]:
```suggestion
All but the individual beatmap rankings can be navigated to by hovering over the `rankings` menu on the website header. For the individual beatmap ranking, they can be found on their beatmap info page.
```

File: wiki/Ranking/en.md
Comment (TPGPL) [inline wiki/Ranking/en.md:17]:
```suggestion
*See also: [Performance points](/wiki/Performance_points)*
```

File: wiki/Ranking/en.md
Comment (TPGPL) [inline wiki/Ranking/en.md:25]:
```suggestion
## Kudosu ranking
```

File: wiki/Ranking/en.md
Comment (TPGPL) [inline wiki/Ranking/en.md:27]:
```suggestion
The Kudosu ranking compares users' accumulated [Kudosu](/wiki/Modding/Kudosu) and lists them accordingly. This ranking requires no playing skills, as Kudosu is gained as a part of the [modding process](/wiki/Modding).
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Ranking/en.md
Comment (MegaApplePi) [reply wiki/Ranking/en.md:25]: @Joehuu suggested to use `kudosu!` (stylized like that) in https://github.com/ppy/osu-wiki/pull/3867#discussion_r527128471, I'm also leaning that way too.... I was about to add it as a term in the ASC to be spelt like that.

File: wiki/Ranking/en.md
Comment (MegaApplePi) [reply wiki/Ranking/en.md:27]: Will apply some of those changes
```suggestion
The kudosu! ranking compares users' accumulated [kudosu!](/wiki/Modding/Kudosu) and lists them accordingly. This ranking requires no playing skills, as Kudosu is gained as a part of the [modding process](/wiki/Modding).
```

File: wiki/Disambiguation/Rank/en.md
Comment (MegaApplePi) [reply wiki/Disambiguation/Rank/en.md:14]: I think this sounds a bit better.

```suggestion
- The verb "to rank", which means going through the [beatmap ranking procedure](/wiki/Beatmap_ranking_procedure).
```

File: wiki/Performance_points/Total_performance_points/en.md
Comment (MegaApplePi) [reply wiki/Performance_points/Total_performance_points/en.md:8]: - [ ] the ppv2 link in this line needs to be fixed, probably missed checking this, should check elsewhere too

**Comment by TicClick** [comment]
what's the current status of this? what input does it need, apart from fixing linting errors? @Pennek and the reviewers

**Comment by Pennek** [comment]
Sorry, ive abandoned this PR. Hope TPGL and Map have an overview

**Comment by MegaApplePi** [comment]
@TicClick I was waiting for an explicit approval from another wikifriend since I began making more changes from the comments/suggestions. Though, I think this pull request should be good enough now since there were at least some more suggestions after a long period of time. I'll go ahead and merge it now.

**Review by MegaApplePi — APPROVED** [review]

**Review by MegaApplePi — APPROVED** [review]

**Pull request merged**
